piksi_pps_sync/package.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>piksi_pps_sync</name>
+  <version>0.0.0</version>
+  <description>Tools to synchronize against the GPS clock.</description>
+
+  <author email="brik@ethz.ch">Rik Bähnemann</author>
+  <maintainer email="brik@ethz.ch">Rik Bähnemann</maintainer>
+  <license>TBD</license>
+
+  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend>catkin_simple</buildtool_depend>
```

Comment:
other packages in this repo do not use catkin_simple. Try to pull in another dependency if possible.

Reply:
Done.